components/server/src/user/user-service.ts

```diff
@@ -419,6 +422,11 @@ export class UserService {
         user.name = user.name || authUser.name || authUser.primaryEmail;
         user.avatarUrl = user.avatarUrl || authUser.avatarUrl;
 
+        if (authUser.created_at && isDateSmaller(authUser.created_at, daysBefore(new Date().toISOString(), 30))) {
```

Comment:
Good catch! 🙏

Reply:
Let's move this check to `createUser`.

Reply:
moved it to the callback that is passed to `createUser` because I needed access to the authUser object. Also looked like a good pace for it otherwise. Fine?